### DISCUSSION TITLE
Feature Request: Watched Repository News or something similar

### DISCUSSION THREAD

**Discussion by asaddon** [community/community#2819] [Mobile] [answered]
Hello, I would like to see a feature where it would show all the **pushed commits** and user interactions for the repository I **Follow/Watch/Star** in a single page, another Github app does have that similar functionality, named [CodeHub](https://github.com/CodeHubApp/CodeHub) and I'm linking the example screenshot from that app below.

[Example Screenshot](https://p20.zdusercontent.com/attachment/66681/2SebdiakzcEQQrXDAwbjOwA8R?token=eyJhbGciOiJkaXIiLCJlbmMiOiJBMTI4Q0JDLUhTMjU2In0..8RrXYyndPBq0k6TVrTVcZw.oKg8yrRBFVuNHNmGKcTcUijkDuQOVA0TOPMEn581HImx-cNUoHip0g0iw-gho9Theb57srdFRxjjw9GS-jjG5Za66cqagAijDpIqlZENToRKDr2PuAWIsQaPsSkeyypVho9fVDSF4VH-bsnE2oEuw9h7VJDoVQ-34BE7xzsvZ5s1ms0naVpogp92_5gSMJbQfUTR1wWt9PrdvmGjI5Z1KEAphAIButwKuHDaF4ozgccYANeeQ6PCl20KTWTUN5K5KPjG3TPURljo2n4ltAseDQ._wF3nDlFJr1AtfHVUaTPUw)

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback! We do offer in-app filters for notifications generated by user interactions: you can visit the Notifications tab, tap on the filter button on the top right corner, scroll down and filter your notifications by Repository. Let us know if this helps!

**Reply by asaddon** [reply]
No, it doesn't help and that's not what I suggested, the filter method doesn't show any commits activity done on the repository.
